solr/solrj/src/java/org/apache/solr/common/cloud/acl/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Classes for managing ACLs to Zookeeper */
+package org.apache.solr.common.cloud.acl;
```

Comment:
Yes this is for backward compatibility. The old classes are left in place, then I created this new package for the new ones. I think it makes sense to have all ZK creds/acl related classes in a separate package, and in the future move the old (non-deprecated classes) to the new package. 

Do you suggest to remove or keep the new package?

As for the name, we need a name to englobe both the notions of creds and ACLs. I choose acl because I think ACL is the goal and the creds are a kind of means. What do you think? I am open to any suggestion.

Reply:
I like creds better, because ACLs are a form of credentials, and so are the ZK Auth stuff.

Let's do it all in one package for now and move to a separate package when we break back-compat